adapters/flipp/flipp.go

```diff
@@ -230,7 +242,20 @@ func buildBid(decision *InlineModel, impId string) *openrtb2.Bid {
 		if decision.Contents[0].Data.Width != 0 {
 			bid.W = decision.Contents[0].Data.Width
 		}
-		bid.H = 0
+		customDataInterface := decision.Contents[0].Data.CustomData
+		customDataMap := customDataInterface.(map[string]interface{})
+
+		bid.H = defaultStandardHeight
+		customDataKey = "standardHeight"
```

Comment:
@MishelleBit looks good you can just move this to the top where you are declaring the `customDataKey`